gateway/events.go

```diff
@@ -76,10 +77,10 @@ type RequestGuildMembersCommand struct {
 	GuildIDs []discord.GuildID `json:"guild_id"`
 	UserIDs  []discord.UserID  `json:"user_ids,omitempty"`
 
-	Query     string `json:"query,omitempty"`
-	Limit     uint   `json:"limit,omitempty"`
-	Presences bool   `json:"presences"`
-	Nonce     string `json:"nonce,omitempty"`
+	Query     option.NullableString `json:"query"`
```

Comment:
Query should have `,omitempty`; it's an optional field: https://discord.com/developers/docs/topics/gateway-events#request-guild-members.